src/linear_codes/mod.rs

```diff
@@ -45,29 +45,29 @@ where
     C: Config,
     H: CRHScheme,
 {
-    /// Get security parameter
+    /// Get the security parameter.
     fn sec_param(&self) -> usize;
 
-    /// Get the inverse of code rate
+    /// Get the distance of the code.
     fn distance(&self) -> (usize, usize);
 
-    /// See whether there should be a well-formedness check
+    /// See whether there should be a well-formedness check.
     fn check_well_formedness(&self) -> bool;
 
-    /// Compute
+    /// Compute the dimensions of the coefficient matrix.
     fn compute_dimensions(&self, n: usize) -> (usize, usize);
 
-    /// Get LeafHash parameters
+    /// Get the hash parameters for obtaining leaf digest from leaf value.
     fn leaf_hash_params(&self) -> &<<C as Config>::LeafHash as CRHScheme>::Parameters;
 
-    /// Get TwoToOneHash paramters
+    /// Get the parameters for hashing nodes in the merkle tree.
     fn two_to_one_params(&self) -> &<<C as Config>::TwoToOneHash as TwoToOneCRHScheme>::Parameters;
 
-    /// Get column hashing parameters
+    /// Get the parameters for hashing a vector of values, representing a column of the coefficient matrix, into a leaf value.
```

Comment:
Weren't we aiming for a hard wrap at 80 chars, including for comments?

Reply:
Yes good catch!